src/_data/sites/objector.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://objector.dev",
+    "name": "Objector",
+    "description": "The personal blog and portfolio of Montique Stevens, a freelance Creative Technologist",
+    "twitter": "nycbeardo",
+    "source_url": "https://github.com/nycbeardo/myeleventyblog"
```

Comment:
@nycbeardo https://github.com/nycbeardo/myeleventyblog seems to 404 for me. Is that a private repo?

Reply:
Yes! Sorry! Changing settings now